script.js

```diff
@@ -1,4 +1,68 @@
+var userWins = 0;
+var computerWins = 0;
+
+var numberGenerator = function () {
+  var randomDecimal = Math.random() * 3;
+  var randomInteger = Math.floor(randomDecimal);
+  var finalNumber = randomInteger + 1;
+  return finalNumber;
+};
+
+var assignMoveToNumber = function () {
+  var generatedNumber = numberGenerator();
+  console.log('number:' + generatedNumber);
+  if (generatedNumber == 1) {
+    return 'scissors';
+  }
+  if (generatedNumber == 2) {
+    return 'paper';
+  }
+  if (generatedNumber == 3) {
+    return 'stone';
+  }
+
+  return 'something went wrong';
+};
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var computersMove = assignMoveToNumber();
+  var myOutputValue = 'enter valid input';
+  console.log('input:' + input);
+  console.log('computer:' + computersMove);
+
+  if (input == computersMove) {
+    myOutputValue = 'DRAW! your move: ' + input + ', computer move: ' + computersMove + '-----User Wins:' + userWins + '-----Computer Wins:' + computerWins;
+  }
+
+  if (input == 'scissors' && computersMove == 'paper') {
```

Comment:
Lots of repeated code below- big no-no! Please use `||` operator between your `&&` boolean statements to re-use code. For example, "if scissors and paper OR paper and stone OR stone and scissors: you win!"